stagemonitor-core/src/main/java/org/stagemonitor/core/CorePlugin.java

```diff
@@ -396,6 +397,33 @@
 			.configurationCategory(CORE_PLUGIN_NAME)
 			.tags("elasticsearch", "advanced")
 			.buildWithDefault(5);
+	private final ConfigurationOption<Collection<String>> remotePropertiesConfigUrls = ConfigurationOption.stringsOption()
+			.key("stagemonitor.configuration.remoteproperties.urls")
+			.dynamic(false)
+			.label("URLs of the remote properties")
+			.description("Must be http or https urls.")
```

Comment:
Elaborate a bit more. Add an example of a spring cloud config url.

Reply:
Extendend the description and provided an example